roles/apicast/templates/apicast-deployment.yaml

```diff
@@ -27,16 +27,22 @@ spec:
         threescale_component_element: gateway
 {% if vars[apicast_env].marin3r.enabled == true %}
         marin3r.3scale.net/status: "enabled"
+<<<<<<< HEAD
```

Comment:
Something weid happened here, there are a few lines shouldnt be here

Reply:
:scream:

Reply:
I guess a I messed up the rebase. It is fixed now.